tests/fixtures/checksum-compare/8.1.0.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<checksum_list>
+	<ps_root_dir version="8.1.0">
+		<dir name="admin">
+			<md5file name=".htaccess">f9801a6607a14db3eebab2da3a649716</md5file>
+			<md5file name="init.php">dc55d3ee8cd557eacc4e191643207698</md5file>
+		</dir>
+		<dir name="app">
```

Comment:
I see a check that has been added on the computation of missing and altered files making the `install/` folder ignored. Could be worth adding a file in this folder and make sure it is not returned by the test?

```suggestion
		</dir>
		<dir name="install">
			<md5file name="setup.php">b0175575517b2a31c3ee8cd55c5b50</md5file>
		</dir>
		<dir name="app">
```